### PR TITLE
Split `cheatcodes.cairo`

### LINKED ISSUE #561: Split `cheatcodes.cairo` into smaller files
e.g. move printing and file operations into `io` module

### PR DESCRIPTION
**Pull request by war-in**
Closes #561

## Introduced changes

- cheatcodes.cairo is splitter into few files

## Checklist

- [x] Linked relevant issue
- [ ] Updated relevant documentation
- [ ] Added relevant tests
- [x] Performed self-review of the code
- [ ] Added changes to `CHANGELOG.md`

### CODE REVIEW

File: snforge_std/src/cheatcodes.cairo
Comment (cptartur) [inline snforge_std/src/cheatcodes.cairo:1]: Should `get_class_hash` be moved to `contract_class.cairo` as well?

File: snforge_std/src/cheatcodes/spoof.cairo
[the remark this answers is not in the document]
Comment (war-in) [reply snforge_std/src/cheatcodes/spoof.cairo:1]: Yeah, good catch

File: snforge_std/src/cheatcodes.cairo
Comment (war-in) [reply snforge_std/src/cheatcodes.cairo:1]: Not sure tbh. It is a standalone feature

**Review by cptartur — APPROVED** [review]

File: snforge_std/src/cheatcodes.cairo
Comment (Radinyn) [reply snforge_std/src/cheatcodes.cairo:1]: I'm with @cptartur on this

**Review by Radinyn — APPROVED** [review]

**Pull request merged**